packages/aws_common/lib/src/js/indexed_db.dart

```diff
@@ -184,19 +198,47 @@ extension PropsIDBObjectStore on IDBObjectStore {
   IDBRequest<void> add(String value, String key) =>
       js_util.callMethod(this, 'add', [value, key]);
 
+  /// Returns an [IDBRequest] object, and, in a separate thread, creates a
+  /// structured clone of the value, and stores the cloned value in the object
+  /// store.
+  ///
+  /// This is for adding new records to an object store created with keyPath set and autoincrement = true
+  IDBRequest<void> push(Map<Object, Object> item) =>
```

Comment:
This is for adding new records to an object store created with keyPath set and autoincrement = true

Otherwise if just calling add with 'value' you get: 
 ` DataError: Failed to execute 'add' on 'IDBObjectStore': A generated key could not be inserted into the value.
`